### ISSUE TITLE
Archiving todo

### ISSUE THREAD

**Issue by evzharko** [CoopBelvedere/laravel-basecamp-api#20]
Please tell me how to implement todo archiving in your library?
Found similar functionality in Recordings, but there is only deletion.
In fact, if you replace
```
sprintf('buckets/%d/recordings/%d/status/trashed.json', $this->bucket, $id) to
sprintf('buckets/%d/recordings/%d/status/archived.json', $this->bucket, $id)
```
then everything is ok. Can you add an archiving method to your lib?

**Comment by yuters** [comment]
Hi !

Sorry, the methods to archive / unarchive are not implemented on the recordings class.
https://github.com/basecamp/bc3-api/blob/master/sections/recordings.md#archive-a-recording

Feel free to send a pull request

**Comment by yuters** [comment]
It's merged and published as v1.3.0 
Note that I've changed the method name to `archive()` and I've also added the `unarchive()` method

Here's the updated docs:
https://github.com/CoopBelvedere/laravel-basecamp-api/blob/master/docs/recordings.md

Thank you! :)
